blocks/api/serializer.js

```diff
@@ -37,8 +39,18 @@ export function getSaveContent( save, attributes ) {
 		}
 	}
 
+	// Adding a generic classname
+	const addClassnameToElement = ( element ) => {
```

Comment:
what about a plugin attribute `class` that can be nullable and if not set default to `wp-block-core-quote`